headers/data/ram.h

```diff
@@ -30,6 +30,7 @@ extern struct team_member_table TEAM_MEMBER_TABLE;
 extern uint32_t FRAMES_SINCE_LAUNCH_TIMES_THREE;
 extern struct sentry_duty SENTRY_DUTY_STRUCT;
 extern bool TURNING_ON_THE_SPOT_FLAG;
+extern struct loaded_attack_sprite_data* LOADED_ATTACK_SPRITE_DATA;
```

Comment:
This struct type isn't define anywhere. Did you mean to define it somewhere?

Reply:
The structure seems to be correctly defined inside `dungeon_mode.h`. Or am I missing something.

Reply:
You're right, not sure how I missed that.